elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -836,6 +836,46 @@ trait RestlasticSearchClientTest {
       }
     }
 
+    "Support queries in filter context" in {
+      val futures = (1 to 10).map { n =>
+        Document(s"queryFilterDoc-$n", Map("f1" -> n, "f2" -> "field2", "f3" -> "field3"))
+      }.map { doc =>
+        restClient.index(index, tpe, doc)
+      }
+
+      val range = Future.sequence(futures)
+      whenReady(range) { _ =>
+        refresh()
+      }
+      refresh()
```

Comment:
is this extra `refresh()` needed?

Reply:
Maybe it isn't, I just copied this part from another test case